modules/ROOT/partials/tools-install.adoc

```diff
@@ -0,0 +1,135 @@
+// tag::cbsbctl[]
+[{tabs}]
+====
+macOS::
++
+--
+. Open a Terminal window and go to the directory where the `cbsbctl` binary is located:
++
+[source,console]
+----
+$ cd couchbase-service-broker-kubernetes_x.x.x-macos_x86_64/
```

Comment:
There is only one package and will untar to `couchbase-service-broker-kubernetes` for all operating systems